index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+ <header> <a href="">My logo</a>
+    <nav> 
+        <a href="#" class="fa fa-bars"></a>
+        </nav>
+        <img class="topim" src="./images/" alt=""></header>
+        
+    <section>
+        <h1>I'm Adrian. <br> Glad to see you </h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <a href="#">LET'S CONNECT</a>
```

Comment:
- [ ] You have a horizontal scrollbar on your web app. Kindly fix the issue. :+1: 

https://github.com/AdrianScharff/My-portfolio-setup/assets/70483835/12dbbe02-5908-42ad-97a0-501d955f0d68